### ISSUE TITLE
Portrait orientation of Barcelona example

### ISSUE THREAD

**Issue by generativearts** [marceloprates/prettymaps#81]
Hello!
Is it possible to make Barcelona example in portrait orientation? Draw map 2/3 proportion rectangle not square.

**Comment by generativearts** [comment]
Thank you

**Comment by danielwiegand** [comment]
Could you maybe provide a short example how to do this with postprocessing?

**Comment by till-m** [comment]
Hey @generativearts & @danielwiegand did you ever figure out how to do this?

**Comment by danielwiegand** [comment]
@till-m No. I ended up generating a quadratic image and changing the aspect ratio by means of Gimp.

**Comment by ga58xes** [comment]
any updates on this?
